usecases/MOC_integration/moc_integration.adoc

```diff
@@ -249,204 +272,203 @@ oc delete all -l app=pfcon
 oc delete route pfcon
 ....
 
-== Running Test Scripts on Openshift
+// == Running Test Scripts on Openshift
```

Comment:
Commenting code out is redundant since we are using git VCS, simply delete what should be removed.